### PR TITLE
Remove Unneeded Task and Update Docs [VS-817]

### PR DESCRIPTION
**Pull request by rsasch**
Generate Avro Files run: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/577e77f2-4174-46c0-9d8a-9fdbbd2a27ed

Generate VDS run: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/27299041-7779-49d7-b7d3-6c0349f5ffc3

Generate VAT run: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/ad1b2687-7df0-4e92-b4e2-b61fcb77cd19

Validate VAT run: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/dae4874f-a619-40d4-84b4-066295b76cb2

### CODE REVIEW

**Review by gbggrant — APPROVED** [review]
LGTM

**Review by RoriCremer — APPROVED** [review]
bless you---you are doing gods work with this pr

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 1]
```diff
@@ -250,10 +244,7 @@ task GenerateSitesOnlyVcf {
         String gcs_subnetwork_name
         Boolean leave_cluster_running_at_end
         String hail_version
-        File? hail_wheel
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:253]: do we wanna keep the hail wheel option in until we are done tweaking the VAT creation pipeline?

Comment (rsasch) [reply scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:253]: There's this ticket to document how to do this (temporarily) for the future, but — according to Kylee and the Hail team — this should not be a permanent part of our pipeline: https://broadworkbench.atlassian.net/browse/VS-1241

**Pull request merged**